tests/js/fixtures/detectors.ts

```diff
@@ -0,0 +1,35 @@
+import {DataConditionGroupFixture} from 'sentry-fixture/dataConditions';
+import {UserFixture} from 'sentry-fixture/user';
+
+import type {Detector} from 'sentry/types/workflowEngine/detectors';
+
+export function DetectorFixture(params: Partial<Detector>): Detector {
+  return {
+    id: '1',
+    name: 'detector',
+    projectId: '1',
+    createdBy: UserFixture().id,
+    dateCreated: new Date('2025-01-01T00:00:00.000Z'),
+    dateUpdated: new Date('2025-01-01T00:00:00.000Z'),
+    lastTriggered: new Date('2025-01-01T00:00:00.000Z'),
```

Comment:
i think the Detector type here is wrong since they'll be strings from the json
```suggestion
    dateCreated: '2025-01-01T00:00:00.000Z',
    dateUpdated: '2025-01-01T00:00:00.000Z',
    lastTriggered: '2025-01-01T00:00:00.000Z',
```

Reply:
Fixed in f5262caffa9d406e4de631e62993f59c4fae35e9 ✨